app/src/store/wallet.ts

```diff
@@ -227,6 +227,7 @@ export default function useWalletStore() {
   const isSupportedNetwork = computed(
     () => (network.value ? DGRANTS_CHAIN_ID === network.value.chainId : true) // assume valid if we have no network information
   );
+  const isCorrectNetwork = computed(() => network.value && DGRANTS_CHAIN_ID === network.value.chainId);
```

Comment:
Created a new one, since `isSupportedNetwork` is not good enough, as we discussed on #261